pom.xml

```diff
@@ -630,6 +631,26 @@
             <artifactId>jackson2-api</artifactId>
             <version>2.10.2</version>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>${slf4j.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>jcl-over-slf4j</artifactId>
+            <version>${slf4j.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>log4j-over-slf4j</artifactId>
+            <version>${slf4j.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-jdk14</artifactId>
+            <version>${slf4j.version}</version>
+        </dependency>
```

Comment:
@olamy Jenkins ships with slf4j and thus these should be provided by Jenkins.  if BlueOcean is bundling them, then this is likely not the correct fix, rather the parent should be bumped (to pickup the jenkins bom and the hpi plugin fixed to avoid packaging libraries incorrectly) or the hpi plugin should be updated.  (or the reason for the inclusion should be otherwise tracked down and fixed?
https://github.com/jenkinsci/jenkins/blob/master/core/pom.xml#L451-L463
https://github.com/jenkinsci/jenkins/blob/master/bom/pom.xml#L108-L134

Reply:
@jtnord good catch. seems weird to mark them `provided` but save some bytes :)